package.json

```diff
@@ -35,4 +36,4 @@
     "prisma": "^5.8.1",
     "sass": "^1.70.0"
   }
-}
+}
```

Comment:
I see the missed empty line at the end of several files. And conflicts with dev branch. Could you update your branch to run linter and fix conflicts.

Reply:
Yes, of course, i will.

Reply:
fixed in subsequent commits